sys-apps/portage/portage-2.3.89-r3.ebuild

```diff
@@ -58,6 +58,7 @@ RDEPEND="
 	xattr? ( kernel_linux? (
 		>=sys-apps/install-xattr-0.3
 	) )
+	zstd? ( app-arch/zstd )
```

Comment:
For https://bugs.gentoo.org/715108 we also want to add a sed call to set BINPKG_COMPRESS="zstd" in cnf/make.globals. Something like this:
```diff
diff --git a/sys-apps/portage/portage-2.3.99-r2.ebuild b/sys-apps/portage/portage-2.3.99-r2.ebuild
index 9944a9afe0f..2ad43c799d3 100644
--- a/sys-apps/portage/portage-2.3.99-r2.ebuild
+++ b/sys-apps/portage/portage-2.3.99-r2.ebuild
@@ -126,6 +126,12 @@ python_prepare_all() {
 			|| die "failed to append to make.globals"
 	fi
 
+	if use zstd ; then
+		einfo "Adding BINPKG_COMPRESS=\"zstd\" to make.globals ..."
+		echo -e '\nBINPKG_COMPRESS="zstd"' >> cnf/make.globals \
+			|| die "failed to append to make.globals"
+	fi
+
 	if use build || ! use rsync-verify; then
 		sed -e '/^sync-rsync-verify-metamanifest/s|yes|no|' \
 			-e '/^sync-webrsync-verify-signature/s|yes|no|' \

```

Reply:
Change made and commit message updated. Tested on 2.3.99-r2.